### PR TITLE
clarify IsBuildJob

### PR DESCRIPTION
**Pull request by lmichael107**

### CODE REVIEW

**Comment by lmichael107** [comment]
@ChristinaLK @agitter either of you could review and merge, given Trent Prall's recent confusion.

File: inter-submit.md [hunk 1]
```diff
@@ -40,6 +40,8 @@ First, download the source code for your software to the submit server.
 Then create the following special submit file on the submit server,
 calling it something like `build.sub`.
 
+Note that you'll want to use `+IsBuildJob = true` to specifically match to CHTC's servers designated for compiling code (which include Matlab compilers and other compilin tools you may need). Compiling servers do not include specialized resources like GPUs, extreme amounts of RAM/disk, etc.; to build/test software in these cases, submit an interactive job without `+IsBuildJob`.
```
Comment (agitter) [inline inter-submit.md:43]:
```suggestion
Note that you'll want to use `+IsBuildJob = true` to specifically match to CHTC's servers designated for compiling code (which include Matlab compilers and other compiling tools you may need). Compiling servers do not include specialized resources like GPUs, extreme amounts of RAM/disk, etc.; to build/test software in these cases, submit an interactive job without `+IsBuildJob`.
```

**Review by agitter — CHANGES_REQUESTED** [review]
This is a helpful addition and would have addressed the recent confusion we saw about build servers.

I corrected one typo but otherwise approve.

File: inter-submit.md (hunk 1 above)
Comment (lmichael107) [reply inter-submit.md:43]: thanks for the typo catch!

**Review by ChristinaLK — APPROVED** [review]

**Pull request merged**